controllers/storagecluster/cephcluster.go

```diff
@@ -690,14 +690,24 @@ func getMinimumNodes(sc *ocsv1.StorageCluster) int {
 	return maxReplica
 }
 
-func getMgrCount() int {
+func getMgrCount(sc *ocsv1.StorageCluster) int {
+	if arbiterEnabled(sc) {
+		return defaults.ArbiterModeMgrCount
+	}
+	mgrCount := sc.Spec.ManagedResources.CephCluster.MgrCount
+	if mgrCount != 0 {
+		return mgrCount
+	}
 	if statusutil.IsSingleNodeDeployment() {
 		return 1
 	}
 	return defaults.DefaultMgrCount
 }
 
-func getMonCount(monCount int, arbiter bool) int {
+func getMonCount(sc *ocsv1.StorageCluster) int {
+	if arbiterEnabled(sc) {
+		return defaults.ArbiterModeMonCount
+	}
 	// return static value if overridden
 	override := os.Getenv(monCountOverrideEnvVar)
```

Comment:
What is this env var used for? Seems like it's not necessary if the spec allows overriding it?

Reply:
Yeah, I also saw this and tried to dig for more details. But just found a 4 year old PR https://github.com/red-hat-storage/ocs-operator/pull/371, where it was added. There is no other mention of this anywhere else in our docs etc. As we will be providing a better way in the CR itself good to remove this.